upgrade_tests/helpers/constants.py

```diff
@@ -7,7 +7,7 @@
 FAKE_REPO_ZOO3 = 'http://inecas.fedorapeople.org/fakerepos/zoo3/'
 FROM_VERSION = os.environ.get('FROM_VERSION')
 TO_VERSION = os.environ.get('TO_VERSION')
-SUPPORTED_SAT_VERSIONS = ['6.2', '6.3', '6.4', '6.5', '6.6', '6.7', '6.8']
+SUPPORTED_SAT_VERSIONS = ['6.2', '6.3', '6.4', '6.5', '6.6', '6.7', '6.8', '6.9']
```

Comment:
@devendra104 are we using these versions `'6.2', '6.3', '6.4', '6.5',` anywhere? I feel good to remove as not worth to keep.

Reply:
No need to remove it, It will help to track all the unsupported version of the respective component.

Reply:
@devendra104 var (SUPPORTED_SAT_VERSIONS) is saying supported version ?

Reply:
This is different and used for some other purposes like variant selection, existence pre, and post-data collection, so please ignore this.

Reply:
okay